frontend/src/v4/modules/sequences/sequences.sagas.ts

```diff
@@ -254,18 +254,7 @@ export function* showSequenceDate({ date }) {
 	// 4 - bond date by sequence start/end date
```

Comment:
bond?

Reply:
That's a typo from another PR, I meant "bound", an in to create an upper(end date) and a lower(start date) bound basically